packages/ceramic-cli/src/bin/ceramic.ts

```diff
@@ -77,10 +78,11 @@ schemas
     .command('create <new-content>')
     .option('--only-genesis', 'Only create the genesis object. No anchor will be created')
     .option('--controllers <controllers>', 'Specify a comma-separated list of the controllers of the schema document. Defaults to' + ' current user')
-    .option('--unique', 'Ensure schema document is unique regardless of content')
+    .option('--unique', 'Ensure schema document is unique regardless of content. This is the default behavior.')
+    .option('--not-unique', 'Creating a schema document with identical content to an existing schema document will be a no-op.')
```

Comment:
Are both of these really needed? Isn't `--not-unique` enough?

Reply:
yes, now it would be.  I guess I was thinking about some combination of backwards compatibility and future-proofing against potential future behavior changes.  But I can remove `--unique` if you prefer just having the one

Reply:
If I remove `--unique` do you still think `--not-unique` is the right name for the remaining flag?  I also thought about naming it something that describes what it *is* rather than what it is not, along the lines of `--global` or `--dedupe`, but went with `--not-unique` for symmetry with `--unique`.  I don't feel strongly either way so if you have a preference LMK

Reply:
@oed, thoughts on removing `--unique` and/or renaming `--not-unique`?

Reply:
Fine with removing `--unique`. Not sure about the global or dedupe names, but fine with renaming if we have a good name. Maybe `--deterministic`?

Reply:
Removed --unique.  I put together a patch to rename the command line flag from `--not-unique` to `--deterministic`, but when I looked at it the lack of symmetry/connection between the command line flag and the `isUnique` DocParam arg seemed weird and likely to confuse users, so I put it back to `--not-unique`, even though I don't love it.

Reply:
Fair enough. I guess another option could be to rename the `isUnique` option to `deterministic` and flip the boolean logic?

Both are fine by me. Latter requires a small change in IDW, but that's easy!

Reply:
Okay, just put up a new version that changes the `isUnique` bool to a `uniquenessMode` enum, which I like a lot better.  The tests will fail until IDW is updated - I'm happy to make the change to IDW just let me know if you like this approach and I'll go ahead and do so.

Reply:
Kind of like the `uniquenessMode` less because it requires devs to import an additional type for a thing that could just be a boolean. Seems a bit excessive. Would love to hear thoughts from @PaulLeCam though!

Reply:
I like the `deterministic` name instead of `isUnique`, especially as the default behavior is changed.
About the `uniquenessMode` it's nice for forwards compatibility if we plan on having more than 2 states here, but if we don't have a good use-case of it a boolean would be simpler.

Reply:
Okay, new version up that puts it back to a bool but names it `deterministic`.  Tests are broken dependent on https://github.com/3box/identity-wallet-js/pull/93